tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockFileReadAllLinesTests.cs

```diff
@@ -149,13 +149,93 @@ public void MockFile_ReadAllLinesAsync_NotExistingFile_ThrowsCorrectFileNotFound
             var mockFileSystem = new MockFileSystem();
 
             var act = new AsyncTestDelegate(async () =>
-                await mockFileSystem.File.ReadAllTextAsync(absentFileNameFullPath)
+                await mockFileSystem.File.ReadAllLinesAsync(absentFileNameFullPath)
             );
 
             var exception = Assert.CatchAsync<FileNotFoundException>(act);
             Assert.That(exception.FileName, Is.EqualTo(absentFileNameFullPath));
             Assert.That(exception.Message, Is.EqualTo("Could not find file '" + absentFileNameFullPath + "'."));
         }
+
+        [Test]
+        public async Task MockFile_ReadLinesAsync_ShouldReturnOriginalTextData()
```

Comment:
`ReadLinesAsync` is not available on all target frameworks, so we need a feature gate for this test. See https://github.com/TestableIO/System.IO.Abstractions/blob/main/tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockFileAppendAllTextTests.cs#L160 for example.

Reply:
Sorry, I wasn't aware of the feature flag.

Reply:
No worries, it's not super intuitive :) Thanks for adapting!